.github/release.yml

```diff
@@ -0,0 +1,23 @@
+changelog:
+  categories:
+    - title: Breaking Changes 🪅
```

Comment:
is this a piñata icon?

Reply:
Yep! :pinata: `:pinata:`

Reply:
I already used :broken_heart: `:broken_heart:` for the label